src/data/events.ts

```diff
@@ -56,30 +56,106 @@ export const EVENTS: Event[] = [
     cover: '/images/events/game-boy.jpg',
   },
   {
-    title: 'Club Project: Part 1',
-    description: 'TBD',
+    title: 'BUGS: Project Part 1',
+    description:
+      'Join us for a workshop where we will learn how to use a REST API to return scraped data.',
     date: new Date('April 14, 2023'),
     startTime: '5 PM',
     endTime: '6 PM',
     location: 'WWH Room 101',
-    cover: '/images/events/tbd.png',
+    cover: '/images/events/bugs.jpg',
   },
   {
     title: 'Club Project: Part 2',
-    description: 'TBD',
+    description:
+      'Join us to learn how to create a React frontend to display our scraped course evaluation data.',
     date: new Date('April 28, 2023'),
     startTime: '5 PM',
     endTime: '6 PM',
     location: 'WWH Room 101',
-    cover: '/images/events/tbd.png',
+    cover: '/images/events/bugs.jpg',
+  },
+  // {
```

Comment:
could you uncomment this event as well?

Reply:
Oops! I didn't see that event in NYU Engage so I didn't uncomment it.